### PR TITLE
[MRG] github & google calendar integrate command

### PR DESCRIPTION
**Pull request by leeeizhang**
Closes https://github.com/MLSysOps/MLE-agent/issues/148

### CODE REVIEW

**Comment by leeeizhang** [comment]
Hi, @huangyz0918. Please test this PR by following steps:

step 1: login your third-party extensions:
```
$ mle integrate
 Which platform do you want to integrate? (Use arrow keys)
 » GitHub
   Google Calendar
```

step 2: test the weekly activities fetch funcions:
```python
from mle.workflow.report import *
get_current_week_github_activities()
get_current_week_google_calendar_activities()
```

Some example outputs:
1. get_current_week_github_activities()
```json
{
   "MLSysOps/MLE-agent":{
      "username":"leeeizhang",
      "period":{
         "start":"2024-08-26",
         "end":"2024-08-31"
      },
      "summary":{
         "total_commits":1,
         "total_pull_requests":1,
         "total_issues":1
      },
      "commits":{
         "count":1,
         "messages":[
            "Merge branch 'MLSysOps:main' into feat/google_calender"
         ]
      },
      "pull_requests":{
         "count":1,
         "details":[
            {
               "number":151,
               "title":"[WIP] github & google calendar integrate command",
               "description":"Closes https://github.com/MLSysOps/MLE-agent/issues/148\r\n",
               "status":"open",
               "commit_count":1,
               "commit_messages":[
                  "add github & google calendar integrate command"
               ]
            }
         ]
      },
      "issues":{
         "count":1,
         "details":[
            {
               "number":151,
               "title":"[WIP] github & google calendar integrate command",
               "description":"Closes https://github.com/MLSysOps/MLE-agent/issues/148\r\n"
            }
         ]
      }
   }
}
```

2. get_current_week_google_calendar_activities()
```json
[
  {
    "summary": "Meeting with xxx",
    "kind": "calendar#event",
    "status": "confirmed",
    "description": "xxx",
    "creator": {
      "email": "xxx@xxx",
      "self": true
    },
    "organizer": {
      "email": "xxx@xxx",
      "self": true
    },
    "htmlLink": "https://www.google.com/calendar/event?eid=NzF2NTBkaHRzaDRsdGo2dXE4OW01NTM5ZjkgbWFnaWNkZXZpbC56aGFuZ0Bt",
    "start_time": "2024-08-29T21:00:00+08:00",
    "end_time": "2024-08-29T22:00:00+08:00"
  }
]
```

**Comment by huangyz0918** [comment]
Tested with success! Thank you for adding this support, great contribution 🎉

**Review by huangyz0918 — APPROVED** [review]

File: mle/cli.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import os
 import yaml
 import click
+import pickle
+import calendar
```
Comment (huangyz0918) [inline mle/cli.py:5]: Seems the calendar and datetime are unused

Comment (leeeizhang) [reply mle/cli.py:5]: yes, I will remove these unused modules :)

Comment (huangyz0918) [reply mle/cli.py:5]: I can help you to do this in my PR, no worries : )

**Pull request merged**